InAppSettingsKit/Models/IASKSpecifier.h

```diff
@@ -29,6 +29,8 @@
 - (id)initWithSpecifier:(NSDictionary *)specifier
         radioGroupValue:(NSString *)radioGroupValue;
 
+- (void)sortIfNeeded;
```

Comment:
I added functionality as an instance method in the name of backwards compatibility. 

If that is not a concern, I would move the sorting into the initializer, but this would requiring injecting the reference to the `settingsReader` in the initializer (in order to to read the localized titles when sorting).